assets/block/build/block.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/block.json",
+  "apiVersion": 2,
+  "name": "wp-contributions/block",
+  "version": "0.1.0",
```

Comment:
Would need to get this version updated to match the intended release version which is 1.2.0

Reply:
thanks a lot for noticing that ! changing it